frontend/src/components/BottomNav.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { StyleSheet, Text, View, Alert } from 'react-native';
+import {BottomStates, GlobalStates} from '../constants/States';
+import { FAB } from 'react-native-paper';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { State, Item } from '../constants/Interfaces';
+import { connect, useDispatch } from 'react-redux';
+import { ActionTypes } from '../constants/ActionTypes';
+import { findItemsWithRoute } from './Helper/AsyncCalls';
+
+interface Props {
```

Comment:
What does the interface syntax do? Is this like an object in other languages?

Reply:
It basically acts like an object (except without any methods). It just denotes what type of variables are in the object specified. If you want to look more into it I suggest reading this: https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html (helped a lot when I was learning)